src/components/probe/index.ts

```diff
@@ -66,6 +66,7 @@ export async function doProbe(
     id: probe.id,
     responseCode: 0,
     url: '',
+    method: 'GET',
```

Comment:
Is the method purposedly prefilled with 'GET' and not read from probe.request[].method?

Reply:
Based on what I read, I think it was originally your code. 
I think what I want to achieve is that we should define the default mLog properties, in this case the method is 'GET'.

Because in the end, after all the probing has been done, it will be passed through the probeBuildLog: